### ISSUE TITLE
Component changes not reflected in Ionic Serve

### ISSUE THREAD

**Issue by t4tapas** [ionic-team/ionic-app-scripts#1491]
**Description:**
The changes done specifically in any on my Ionic custom Components are not loading in Ionic Serve. I know this issue is posted before and tried everything but the issue is still is there. 
I have created created few Ionic custom Component as
`$ ionic generate component MyComponent`
It was working just fine for last 1 month and there was no changes to those component TS files.
Few days back I hade to change some code in one of the Component TS file.

A method inside the Component TS was calling another method of a Provider. I changed the number of parameters in the signature of the Provider from 2 to 1 and changed the Component method accordingly. Its still showing the error that my component method has 2 parameter and trying to calling my Provider method with 1 parameter.

Any other changes done in Pages or Provider are fine. Just the component TS file changes are not reflected.

**Steps to Reproduce:**
Tried runing in browser and emulators.
Tried `npm run build`

**Output:**
TypeScript error for the old code.

**My `ionic info`:**
```
@ionic/cli-utils  : 1.15.2
ionic (Ionic CLI) : 3.15.2
```
global packages:

`cordova (Cordova CLI) : 7.1.0`
local packages:

```
@ionic/app-scripts : 3.0.1
Cordova Platforms  : android 6.3.0 ios 4.5.2
Ionic Framework    : ionic-angular 3.6.0
```
System:

```
ios-deploy : 1.9.1
ios-sim    : 5.0.13
Node       : v6.11.2
npm        : 3.10.10
OS         : macOS Sierra
Xcode      : Xcode 9.0.1 Build version 9A1004
```

**Comment by aravindvakil** [comment]
> @raxkaynan It is working now. I just followed the steps on this link:
> https://angular-meteor.com/tutorials/whatsapp2/ionic/setup
> And my code comes to work again with live reload.
> I think it was a mistake in webpack-merge plugin I used to customize ionic webpack.config.js file.

Thanks a lot. It's detecting changes

**Comment by MichaelWRitchie** [comment]
Having the same problem. HTML doesn't update

Ionic:

   ionic (Ionic CLI)  : 4.10.1 (C:\Users\Micha\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.2.2

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : not available
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 1.2.1, (and 5 other plugins)

System:

   NodeJS : v10.15.1 (C:\Program Files\nodejs\node.exe)
   npm    : 6.2.0
   OS     : Windows 10

**Comment by TheOnceAndFutureSmalltalker** [comment]
Same symptom, but different cause.  With me, file watching in VSCode was using up all of the file watching handles - and unnecessarily.  In my app I had added the ios and android platforms which added many, many files not really for editing, but were still consuming file watching handles.  In my VSCode settings, I added to the file watcher exclude list to exclude the /platforms folder and /plugins folder and restarted VSCode.  Now, there were plenty of file watcher handles for the server to use and notice changes taking place in the editor and recompile the app.
